lnd_test.go

```diff
@@ -3115,6 +3412,10 @@ var testsCases = []*testCase{
 		name: "revoked uncooperative close retribution",
 		test: testRevokedCloseRetribution,
 	},
+	{
+		name: "revoked uncooperative close retribution post breach conf",
+		test: testRevokedCloseRetributionPostBreachConf,
```

Comment:
This set and the one above deem to have a good bit of duplication within their test code, and also of their assertions within the tests. What's the rationale of creating an entirely new test? Why not just roll this new one into the original test case?